.eslintignore

```diff
@@ -39,6 +39,33 @@ packages/lit-element/lit.min.*
 packages/lit-element/platform-support.*
 packages/lit-element/private-ssr-support.*
 
+packages/lit-element-starter-ts/node_modules/
+packages/lit-element-starter-ts/lib/
+packages/lit-element-starter-ts/test/
+
+# top level source
+packages/lit-element-starter-ts/**/my-element.js
+packages/lit-element-starter-ts/**/my-element.js.map
+packages/lit-element-starter-ts/**/my-element.d.ts
+packages/lit-element-starter-ts/**/my-element.d.ts.map
+
+packages/lit-element-starter-ts/node_modules/*
+packages/lit-element-starter-ts/docs/*
+packages/lit-element-starter-ts/docs-src/*
+packages/lit-element-starter-ts/**/rollup-config.js
+packages/lit-element-starter-ts/**/custom-elements.json
+
+packages/lit-element-starter-js/node_modules/
+
+# only generated for size check
+packages/lit-element-starter-js/**/my-element.bundled.js
+
+packages/lit-element-starter-js/node_modules/*
```

Comment:
Is this needed since your'e ignoring the whole folder?

Reply:
Same comment above.